steam_wrapper/steam_wrapper.py

```diff
@@ -351,6 +353,11 @@ def configure_shared_library_guard():
     else:
         os.environ["LD_AUDIT"] = f"/app/links/$LIB/libshared-library-guard.so"
 
+def clear_empty_environment(current_info):
+    environment = current_info["environment"]
+    for key, value in os.environ.items():
+        if environment.get(key) == "":
+            del os.environ[key]
```

Comment:
Maybe spit a debug log here just in case?

Reply:
Added logging, hopefully one that will not confuse the end user.